test/base_results/upgrades/annual/results_output.csv

```diff
@@ -14,11 +14,11 @@ project_national-All Upgrades-0002.osw,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,74.59
 project_national-Ceilings-0003.osw,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 project_national-Ceilings-0004.osw,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-project_national-Ceilings-0005.osw,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+project_national-Ceilings-0005.osw,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,114.02,408.98,23.85,572.5,3400.88,,,368.18,38.89,,3259.2,,1880.28,,,253.89,121.51,2788.96,28.37,3045.9,,,434.85,499.71,17690.16,450.2,,,,,,,,,,,,,,17690.16,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.325,1.365,0.08,1.81,10.786,0.0,0.0,0.0,1.091,0.0,0.102,0.0,8.572,0.0,0.0,5.674,0.0,0.0,0.713,0.341,8.048,0.075,0.0,0.0,8.988,0.0,0.0,0.0,1.384,1.481,0.0,1.287,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,52.121,52.121,0.0,52.121,52.121,0.0,0.0,0.0,0.0,0.0,305.7,0.0,2883.3,7506.7,33938.0,0.0,42743.0,997.0,4615.0,800.0,6411.0,1606.0,126.0,8615.0,0.0,2949.0,2520.0,0.0,0.0,0.0,24581.0,2999.0,5766.0,1281.0,145.0,9772.0,0.0,11383.0,0.0,0.0,8794.0,41406.0,4376.0,5656.0,91.4,33.8,34.003,8.68,4.698,0.0,0.0,0.974,6285.4,10652.3,21.664,28.799,0.0,0.0,1853.77,144.0,,1997.77,,,,,,,,1997.77,False,20.0,697.28,2179.0,65370.0,2179.0,0.0,2179.0,2755.0,0.0,7190.7,999.0,Insulation Ceiling|R-60,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0.0,3080.2,33.94,0.0,42.74,0.0,50.0,171.3,7190.7,1658.6,2265.2,0.0,205.6,15.51,11.64,15.66,3.5,4.4,2.89,0.82,0.92,0.48,15.0,12.64,5.4,7.75,101.0,10.65
```

Comment:
Looks like we certainly increase to 3 out of 5 samples getting the ceiling insulation upgrade. 👍